src/tools/compiletest/src/runtest.rs

```diff
@@ -1725,6 +1725,17 @@ impl<'test> TestCx<'test> {
         }
     }
 
+    fn use_dynamic_linking(&self) -> bool {
+        if self.config.target.contains("vxworks") {
+            match env::var("DYLINK") {
```

Comment:
How about  "RUST_TEST_DYLIB" (there are already a couple of other RUST_TEST_... environment variables).

Reply:
or RUST_TEST_DYLINK.

Reply:
I think RUST_TEST_DYLINK is better.